### PR TITLE
Fix missing call to `useRcControlsConfig()`

### PR DESCRIPTION
**Pull request by digitalentity**
Fixes https://github.com/iNavFlight/inav/issues/1182

### CODE REVIEW

File: src/main/fc/config.c [hunk 1]
```diff
@@ -429,6 +429,8 @@ static void activateConfig(void)
     activateControlRateConfig();
 
     resetAdjustmentStates();
+    
+    useRcControlsConfig();
```
Comment (martinbudden) [inline src/main/fc/config.c:433]: This function should probably be renamed now.

**Comment by digitalentity** [comment]
probably `updateRcControlsConfig`?

File: src/main/fc/config.c (hunk 1 above)
Comment (digitalentity) [reply src/main/fc/config.c:433]: Probably `updateRcControlsConfig`?

Comment (martinbudden) [reply src/main/fc/config.c:433]: Well, I think we should avoid using the term "config". How about `rcUpdateModeActivationConditions`?

Comment (digitalentity) [reply src/main/fc/config.c:433]: It gives false impression that function actually *updates* the mode activation conditions while it actually only processes them and caches some flags.
`updateUsedModeActivationConditionFlags()` ?

Comment (martinbudden) [reply src/main/fc/config.c:433]: OK, that is a better name.

Comment (martinbudden) [reply src/main/fc/config.c:433]: Addressed by PR https://github.com/iNavFlight/inav/pull/1251

**Pull request merged**